src/safeposix/syscalls/net_calls.rs

```diff
@@ -1765,22 +1765,102 @@ impl Cage {
         return 0;
     }
 
+    /// ### Description
```

Comment:
great job on providing detailed comments! what about the tests? do we have tests for these syscalls?